gammapy/utils/fitting/iminuit.py

```diff
@@ -0,0 +1,67 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""iminuit fitting functions.
+"""
+from __future__ import absolute_import, division, print_function, unicode_literals
+import numpy as np
+from ..modeling import ParameterList, Parameter
+
+
+__all__ = [
+    'fit_minuit',
+]
+
+
+def fit_minuit(parameters, function):
```

Comment:
Return Minuit object also.
Needed for power users and debugging.

Reply:
:+1: